pkg/modules/replication.go

```diff
@@ -71,6 +71,14 @@ var ReplicationSupportedDrivers = map[string]SupportedDriverParam{
 		PluginIdentifier:              drivers.PowerScalePluginIdentifier,
 		DriverConfigParamsVolumeMount: drivers.PowerScaleConfigParamsVolumeMount,
 	},
+	"powerflex": {
```

Comment:
Do we need to have both PowerFlex and vxflexos cases?

Reply:
Yes we need to have both